bin/validate-peptide-table.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+"""
+check validity of input provided peptide table
+"""
+
+# TODO is there a way to import the $baseDir variable?
```

Comment:
You can pass in `$baseDir` from Nextflow to Python as part of `sys.argv`. That said, I don't grok the complete picture quite well enough to comment on whether that would be helpful.